Makefile

```diff
@@ -1,12 +1,86 @@
 mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
 mkfile_dir := $(dir $(mkfile_path))
-TESTS_DIR ?= "$(mkfile_dir)tests"
-RESOURCES_DIR ?= "$(mkfile_dir)resources"
+TESTS_DIR ?= $(mkfile_dir)tests
+RESOURCES_DIR ?= $(mkfile_dir)resources
+TIMEOUT ?= 5m
+CONTROLLER_CHART ?= kubewarden/kubewarden-controller
+NAMESPACE ?= kubewarden
+K3D_VERSION ?= v5.0.0
 
-.PHONY: runall
-runall:
-	RESOURCES_DIR=$(RESOURCES_DIR) bats \
+KUBEWARDEN_CONTROLLER_CHART_VERSION ?= 0.3.3
+KUBEWARDEN_CONTROLLER_CHART_RELEASE ?= kubewarden-controller
+KUBEWARDEN_CRDS_CHART_VERSION ?= 0.1.1
```

Comment:
I think it would be nice to infer this default version from the version specified in the Chart.yaml